packages/cozy-stack-client/src/NextcloudFilesCollection.js

```diff
@@ -226,6 +233,21 @@ class NextcloudFilesCollection extends DocumentCollection {
     }
     throw new FetchError(resp, resp.json())
   }
+
+  /**
+   * Copy a file or folder to another path on the same Nextcloud server
+   *
+   * @param {object} file - The `io.cozy.remote.nextcloud.files` file to copy
+   * @param {object} to - Whether to copy the file
+   */
+  async copy(file, to) {
+    return await this.stackClient.fetchJSON(
+      'POST',
+      `/remote/nextcloud/${file.cozyMetadata.sourceAccount}/copy/${file.path}${
+        to ? `?Path=${to.path}/${file.name}` : ''
```

Comment:
What happens if there is no "to" destination?

Reply:
The file will be copied into the same folder as the file being copied